server/src/resolvers/Newbie.js

```diff
@@ -0,0 +1,95 @@
+function buddy(parent, args, context) {
+  return context.prisma.newbie({ id: parent.id }).buddy();
+}
+
+async function newbieTasks(parent, args, context) {
+  return await context.prisma.newbie({ id: parent.id }).newbieTasks();
+}
+
+async function buddyTasks(parent, args, context) {
+  return await context.prisma.newbie({ id: parent.id }).buddyTasks();
+}
+
+async function newbieCompleted(parent, args, context) {
+  const newbieCompletedTasks = await context.prisma
+    .newbie({ id: parent.id })
+    .newbieTasks({
+      where: {
+        status: 'COMPLETED',
+      },
+    });
+  return newbieCompletedTasks.length;
+}
+
+async function newbieUncompleted(parent, args, context) {
+  const newbieUncompletedTasks = await context.prisma
+    .newbie({ id: parent.id })
+    .newbieTasks({
+      where: {
+        status: 'UNCOMPLETED',
+      },
+    });
+  return newbieUncompletedTasks.length;
+}
+
+async function buddyCompleted(parent, args, context) {
+  const buddyCompletedTasks = await context.prisma
+    .newbie({ id: parent.id })
+    .buddyTasks({
+      where: {
+        status: 'COMPLETED',
```

Comment:
Could be replaced with constant `STATUS.COMPLETED` from https://github.com/BuildIt-Poland/buddy-app/blob/67d406696f7d1e3e70f96165bf2bdeda8465abc4/server/src/utils.js#L6 following DRY.

Just like you did in `Mutation.js` 
https://github.com/BuildIt-Poland/buddy-app/blob/67d406696f7d1e3e70f96165bf2bdeda8465abc4/server/src/resolvers/Mutation.js#L163

Same occurs in lines `51` and `40` an `18`